app/pages/sections/catalogue-solutions/order-item/flat/patient/ui-tests/withoutSavedData.ui.test.js

```diff
@@ -80,14 +57,13 @@ const mocks = () => {
 const pageSetup = async (withAuth = true, postRoute = false) => {
   if (withAuth) {
     mocks();
-    await setCookies();
-    await selectedRecipientIdState();
-    await selectedSolutionIdState();
-    await selectedPriceIdState();
-    await serviceRecipientNameState();
+    await setState('fakeToken', authTokenInSession);
+    await setState('selectedRecipientId', selectedRecipientIdInSession);
+    await setState('selectedRecipientName', selectedRecipientNameInSession);
+    await setState('selectedSolutionId', solutionIdInSession);
+    await setState('selectedPriceId', selectedPriceIdInSession);
     if (postRoute) {
-      await solutionNameState();
-      await selectedPriceState(selectedPrice);
+      await setState('orderItemPageData', orderItemPageDataInSession);
```

Comment:
This does neaten it up 👍  Question did come to mind though, do we need anywhere that clears this information. As a user I create a new catalogue solution order item. The go back to the "newsolution" address by hand, will it bring up the data in session? Obviously edge case, so just something to consider if not already.

Reply:
That is a good point and something we haven't really considered just yet. We do need to figure out an elegant way of clearing the relevant keys from session at the right time.